plugins/catalog-backend/src/stitching/Stitcher.ts

```diff
@@ -105,26 +99,32 @@ export class Stitcher {
         errors: 'refresh_state.errors',
         incomingReferenceCount: 'incoming_references.count',
         previousHash: 'final_entities.hash',
-        relationType: 'relations.type',
-        relationTarget: 'relations.target_entity_ref',
       })
       .from('refresh_state')
       .where({ 'refresh_state.entity_ref': entityRef })
       .crossJoin(this.database.raw('incoming_references'))
       .leftOuterJoin('final_entities', {
         'final_entities.entity_id': 'refresh_state.entity_id',
+      });
+
+    const relationsResult: Array<{
+      relationType: string;
+      relationTarget: string;
+    }> = await this.database
```

Comment:
Was gonna write about potential consistency problems here, but thinking another round about it I think we're good. If there happens to be a conflicting write that comes in-between these two queries, then we'd be guaranteed that that eventually triggers another stitching, quickly correcting the mistake.

As long as ^ is true I'm thinking we should run these two queries in parallel to save a bit of extra time, since it's not in the same transaction anyway.